src/qnngds/devices.py

```diff
@@ -273,3 +271,143 @@ def snspd_vert(
 
     D.info = S.info
     return D
+
+def resistor_meander(
+    layer: int = 1,
```

Comment:
might be silly but a bit like in Phidl, can you actually put the layer param at the end of the argument list?